gnucash/report/report-system/test/CMakeLists.txt

```diff
@@ -29,6 +33,10 @@ set(GUILE_DEPENDS
   )
 GNC_ADD_SCHEME_TESTS(${scm_test_report_system_SOURCES})
 
+if (HAVE_SRFI64)
+  gnc_add_scheme_tests ("${scm_test_report_system_with_srfi64_SOURCES}")
+endif (HAVE_SRFI64)
```

Comment:
Are there environments we support that don't have SRFI64 ?

Reply:
erm the travis-ci ubuntu one doesn't have SRFI64. i've copied from test-transaction.

Reply:
Ah indeed. Ubuntu 14.04 ships guile 2.0.9 and srfi-64 was introduced in 2.0.10. So there we go, we need the conditional...